packages/core/src/util/txInspector.ts

```diff
@@ -0,0 +1,113 @@
+import {
+  Address,
+  CertificateType,
+  Lovelace,
+  StakeAddressCertificate,
+  StakeDelegationCertificate,
+  TxAlonzo,
+  Value
+} from '../Cardano';
+import { BigIntMath } from './BigIntMath';
+import { coalesceValueQuantities } from '../Cardano/util';
+import { isAddressWithin, isOutgoing } from '../Address/util';
+
+type Inspector<Inspection> = (tx: TxAlonzo) => Inspection;
+type Inspectors = { [k: string]: Inspector<unknown> };
+type TxInspector<T extends Inspectors> = (tx: TxAlonzo) => {
+  [k in keyof T]: ReturnType<T[k]>;
+};
+
+// Inspectors results types
+export type SendReceiveValueInspection = Value;
+export type DelegateInspection = StakeDelegationCertificate[];
```

Comment:
```suggestion
export type DelegationInspection = StakeDelegationCertificate[];
```
Sorry, I missed this one on the first review

Reply:
done in [c1a8d4f](https://github.com/input-output-hk/cardano-js-sdk/pull/224/commits/c1a8d4fb1567f0971a5291cefa0eccf3c2a125bd)